GliaWidgetsTests/Lib/Coordinator/UIViewControllerTests.swift

```diff
@@ -0,0 +1,19 @@
+@testable import GliaWidgets
+import XCTest
+
+class UIViewControllerTests: XCTestCase {
+    func test_deinit() {
+        var vc1 = UIViewController()
+        var didDeinit = false
+
+        vc1.onDeinit = {
+            didDeinit = true
+        }
+
+        vc1 = UIViewController()
+
+        DispatchQueue.main.async {
```

Comment:
Is it possible to get rid of `DispatchQueue.main.async` here?

Reply:
Fixed